src/composables/useWebExtensionStorage.ts

```diff
@@ -132,7 +132,14 @@ export function useWebExtensionStorage<T>(
     data,
     async () => {
       try {
-        await (data.value == null ? storageInterface.removeItem(key) : storageInterface.setItem(key, await serializer.write(data.value)))
+        if (data.value == null) {
+          storageInterface.removeItem(key)
```

Comment:
Noticed that promises no longer get `await`ed. Not sure how big of a deal that is.